lua/neogit/lib/input.lua

```diff
@@ -0,0 +1,63 @@
+local M = {}
+
+if not _G.neogit then _G.neogit = {} end
```

Comment:
`neogit` is in my opinion a bad name for a global library variable. Maybe prefixing the variable with 2 `_` will make it less likely to be used by users.